src/dmd/expressionsem.d

```diff
@@ -4570,9 +4570,7 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
 
                 //printf("tf = %s, args = %s\n", tf.deco, (*arguments)[0].type.deco);
                 .error(exp.loc, "%s `%s%s` is not callable using argument types `%s`",
-                    p, exp.e1.toChars(), parametersTypeToChars(tf.parameters, tf.varargs), buf.peekString());
-                if (failMessage)
-                    errorSupplemental(exp.loc, failMessage);
+                    p, exp.e1.toChars(), parametersTypeToChars(tf.parameterList.parameters, tf.varargs), buf.peekString());
```

Comment:
ditto

Reply:
??

Reply:
"This doesn't look like a refactoring change"

Reply:
Oh, I see. I accidentally deleted a couple lines.